dispatcher.py

```diff
@@ -93,20 +93,20 @@ def run(self):
                     self.sendGo(True)
 
                 if len(self.bots) >= 2:
-                    logging.info("Game can be played.....")
+                    logging.info('Game can be played.....')
                     self.threads.append(self.createBattle(self.games, self.server))
                     self.games += 1
                     self.bot_id = 0
             except OSError as e:
-                logging.info(f"OSError Exception: {e.strerror}")
+                logging.info(f'OSError Exception: {e.strerror}')
         for t in self.threads:
             t.join()
 
     def logBots(self):
         data = self.server.get_data()
         cData = copy.copy(data)
         for k, v in cData.items():
```

Comment:
k, v - that is not the best name for variables :)
I know it means key, value but I have no idea what they store.

Reply:
improved.